MBBSEmu.Tests/ExportedModules/Majorbbs/register_module_Tests.cs

```diff
@@ -35,21 +35,21 @@ public void register_module_test()
             ExecuteApiTest(HostProcess.ExportedModules.Majorbbs.Segment, REGISTER_MODULE_ORDINAL, new List<FarPtr> { modulePointer });
 
             Assert.Equal(Encoding.ASCII.GetString(moduleStruct.descrp).TrimEnd('\0'), mbbsModule.ModuleDescription);
-            Assert.Equal(new FarPtr(0x0001, 0x0001), mbbsModule.EntryPoints["sttrou"]);
-            Assert.Equal(new FarPtr(0x0002, 0x0002), mbbsModule.EntryPoints["dlarou"]);
-            Assert.Equal(new FarPtr(0x0003, 0x0003), mbbsModule.EntryPoints["finrou"]);
-            Assert.Equal(new FarPtr(0x0004, 0x0004), mbbsModule.EntryPoints["huprou"]);
-            Assert.Equal(new FarPtr(0x0005, 0x0005), mbbsModule.EntryPoints["injrou"]);
-            Assert.Equal(new FarPtr(0x0006, 0x0006), mbbsModule.EntryPoints["lofrou"]);
-            Assert.Equal(new FarPtr(0x0007, 0x0007), mbbsModule.EntryPoints["lonrou"]);
-            Assert.Equal(new FarPtr(0x0008, 0x0008), mbbsModule.EntryPoints["mcurou"]);
-            Assert.Equal(new FarPtr(0x0009, 0x0009), mbbsModule.EntryPoints["stsrou"]);
-            Assert.NotEqual(FarPtr.Empty, mbbsEmuMemoryCore.GetPointer(mbbsEmuMemoryCore.GetVariablePointer("MODULE") + (2 * mbbsModule.StateCode)));
+            Assert.Equal(new FarPtr(0x0001, 0x0001), mbbsModule.ModuleDlls[0].EntryPoints["sttrou"]);
```

Comment:
a property such as `PrimaryModuleDll` or `MainModuleDll` would be useful here

Reply:
Added property `MainModuleDll` to `MbbsModule` and updated references to `[0]` to make them easier to understand